crates/cxx-qt-build/src/lib.rs

```diff
@@ -495,13 +496,24 @@ impl CxxQtBuilder {
             .expect("Could not find Qt installation");
         qtbuild.cargo_link_libraries(&mut self.cc_builder);
 
+        // Allow for Qt 5 or Qt 6 as valid values
+        println!("cargo:rustc-check-cfg=cfg(cxxqt_qt_version_major, values(\"5\", \"6\"))");
         // Find the Qt version and tell the Rust compiler
         // this allows us to have conditional Rust code
         CxxQtBuilder::define_cfg_variable(
             "cxxqt_qt_version_major".to_string(),
             Some(qtbuild.version().major.to_string().as_str()),
         );
 
+        // Seed all values from Qt 5.0 through to Qt 7.99
+        for major in 5..=7 {
+            println!("cargo:rustc-check-cfg=cfg(cxxqt_qt_version_at_least_{major})");
```

Comment:
Maybe put all the rustc-check-cfg calls into a single function, so that we have a central place that defines all possible cfg flags?

Reply:
Attempted to make a method which escapes values etc